data/v2/messages.json

```diff
@@ -46,11 +46,11 @@
               "justify_qc": {
                 "_pd": null,
                 "data": {
-                  "leaf_commit": "COMMIT~tqpBNqqt63akxJWcfI46uDV8c_Sn_aZ777yCj4X_ukDG"
```

Comment:
This is the genesis leaf commit, which is different now because we use the base version as the genesis version when creating this message, instead of using V1 for the genesis header